src/Microsoft.DotNet.Interactive.Documents/ParserServer/NotebookParserServer.cs

```diff
@@ -25,13 +25,13 @@ public NotebookParserServer(TextReader input, TextWriter output)
 
         public static KernelInfoCollection WellKnownKernelInfos = new()
         {
-            new("csharp", new[] { "c#", "C#", "cs" }),
-            new("fsharp", new[] { "f#", "F#", "fs" }),
-            new("pwsh", new[] { "powershell" }),
-            new("javascript", new[] { "js" }),
-            new("html"),
-            new("sql"),
-            new("kql"),
+            new("csharp", languageName: "csharp", aliases: new[] { "c#", "C#", "cs" }),
+            new("fsharp", languageName: "fsharp", aliases: new[] { "f#", "F#", "fs" }),
+            new("pwsh", languageName: "powershell", aliases: new[] { "powershell" }),
+            new("javascript", languageName: "javascript", aliases: new[] { "js" }),
+            new("html", languageName: "html"),
+            new("sql", languageName: "sql"),
+            new("kql", languageName: "kql"),
```

Comment:
These should use the canonical language names.

Reply:
How so, do you mean casing?  E.g., "JavaScript", "HTML", "SQL", "KQL"?

Reply:
Yes, and `C#`, not `csharp`.